<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendSign</class>
 <widget class="QWidget" name="SendSign">
  <layout class="QVBoxLayout" name="verticalLayoutPage" stretch="0,1,0,0">
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
     <widget class="QFrame" name="frameConfirmTransaction">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutFrame">
       <property name="alignment">
         <enum>Qt::AlignTop</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutConfirmTransaction" stretch="0,0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutConfirmTransaction">
           <item>
            <widget class="QLabel" name="labelConfirmTransaction">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Confirm Transaction</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
           <widget class="QLabel" name="labelAreYouSure">
             <property name="text">
               <string></string>
             </property>
           </widget>
         </item>
         <item>
          <widget class="QLabel" name="explanation">
           <property name="text">
            <string></string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalSectionControls" stretch="0,0,0,0">
           <property name="spacing">
            <number>8</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
            <item>
             <widget class="QLabel" name="questionLabel">
             </widget>
            </item>
          </layout>
         </item>
       </layout>
     </item>
     </layout>
     </widget>
   </item>
   <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string></string>
        </property>
        <property name="text">
         <string></string>
        </property>
        <property name="icon">
         <iconset resource="../bitcoin.qrc">
          <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Edit transaction</string>
        </property>
        <property name="text">
         <string>&amp;Edit</string>
        </property>
        <property name="icon">
         <iconset resource="../bitcoin.qrc">
          <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Abort fee bump</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="../bitcoin.qrc">
          <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
 </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
